pkg/rules/remotewrite/remotewrite.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package remotewrite
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/storage/remote"
+	"gopkg.in/yaml.v2"
+)
+
+// Config represents a remote write configuration for Thanos stateless ruler.
+type Config struct {
+	Name        string                    `yaml:"name"`
```

Comment:
Why do we need `name`? Where is it used?

Reply:
Can we remove this `Config` struct and use the `RemoteWriteConfig` from Prometheus directly?

Reply:
> Can we remove this `Config` struct and use the `RemoteWriteConfig` from Prometheus directly?

Right, and `RemoteWriteConfig` already has the `name` field for it to use.

Reply:
Oh, I see. Thanks for the clarification.